grep-9999-correctiq.md

```diff
@@ -0,0 +1,57 @@
+# GREP [XXXX] -- Upstreaming gr-correctiq
+
+- Original Author: Michael Piscopo <ghostop14@gmail.com>
+- Champion: Michael Piscopo <ghostop14@gmail.com>
+- Status: Draft
+
+History:
+- 07-Jan-2020: Initial Draft
+
+## Abstract
+
+The DC spike created by the IQ sampling process has long been an issue with the sampling 
+process. Software such as GQRX and SDRSharp have built-in algorithms that can be enabled 
+to remove this artifact.  However, GNU Radio has no built-in blocks to address 
+this feature.  A second related feature common in receiver software is the ability 
+to correct for inverted spectrum.  The gr-correctiq OOT module provides blocks that 
+address both of these issues.  Therefore, this GREP is to propose that gr-correctiq 
+be merged into GNU Radio core to make these solutions immediately available to users 
+without the need to install the OOT modules.
+
+## Copyright / License
+
+[I could use a suggestion here.  I'd like to retain the right to use the code as I 
```

Comment:
This is the copyright for the GREP itself, not the code that will come out of it. You can copy the line from other GREPs, if you like. It won't affect the code.
Note that the CLA you signed for the code itself will let you retain use of the code (i.e., after it goes into GNU Radio). You don't give up ownership when submitting code, you just can't force GNU Radio to remove your code again in the future.